### PR TITLE
chore(text): update text content

### PR DESCRIPTION
**Pull request by marktnoonan**
Hi! I noticed some minor typos and inconsistent text capitalization, as well as what looks like flipped `help` and `description` text in one spot for the `landmark-unique` rule. This PR fixes the ones I was able to spot.

I noticed some overlap with https://github.com/dequelabs/axe-core/issues/4385, I'd be happy to grab the text changes mentioned there in this PR as well, ("Ensures" -> "Ensure" in rule descriptions), if that's not stepping on anybody's toes.

### CODE REVIEW

**Review by WilcoFiers — CHANGES_REQUESTED** [review]
Thank you very much @marktnoonan. Would you mind making these changes in their source files instead of in the generated file though?

Sorry about that, we normally put a comment at the top of generated files. Can't put comments in JSON unfortunately.

File: locales/_template.json
[the remark this answers is not in the document]
Comment (marktnoonan) [reply locales/_template.json:1]: Oh that makes so much sense, yes I'll move changes to the proper place, thanks!

File: doc/rule-descriptions.md [hunk 1]
```diff
@@ -86,9 +86,9 @@
```
Comment (marktnoonan) [inline doc/rule-descriptions.md:86]: @WilcoFiers This file is also generated - should I remove these changes from the PR also?

Comment (WilcoFiers) [reply doc/rule-descriptions.md:86]: No, we try to keep this in sync with the .json files in the repo. We actually need the _template.json file updated too. Would you mind running `npm run build` to update that file as well?

Comment (marktnoonan) [reply doc/rule-descriptions.md:86]: Ah understood - we want the generated files updated and committed, but they are not edited directly, and so they stay current with the values used elsewhere in code. Updated the PR with the generated _template file.

Comment (marktnoonan) [reply doc/rule-descriptions.md:86]: @WilcoFiers I believe everything is committed and good to go at this point.

**Review by WilcoFiers — APPROVED** [review]
LGTM. Thank you for the kind contribution @marktnoonan!

For the books; I reviewed this PR for security.

**Pull request merged**